tests/v2.test.js

```diff
@@ -324,27 +338,28 @@ const tests = (client) => {
             ],
             supports: [
               'cors',
+              'jsonldMediaType',
             ],
           },
         ],
       }, requiredJson);
 
-      const response = await client.get(`${prefix}/${identifier}/info.json`);
+      const response = await client({ uri: `${prefix}/${identifier}/info.json`, headers });
 
       expect(response.statusCode).toBe(200);
-      expect(response.headers['content-type']).toBe('application/json');
+      expect(response.headers.vary).toBe('Accept, X-Test-Run');
```

Comment:
Can probably strip `X-Test-Run` in `deliver` at the edge.

Reply:
In fact don't receive it directly from the shield (as it's stripped by Fastly) already.